manifests/w/WowUp/WowUp/Beta/2.21.0-beta.6/WowUp.WowUp.Beta.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: WowUp.WowUp.Beta

```

```suggestion
PackageIdentifier: WowUp.Wowup.Beta
```

https://github.com/microsoft/winget-pkgs/tree/master/manifests/w/WowUp/Wowup has a lower-case letter (u vs U)